textrepo-app/src/main/java/nl/knaw/huc/service/task/indexer/JdbiIndexFileTaskBuilder.java

```diff
@@ -135,6 +140,7 @@ private TypesDao types() {
     }
 
     private void indexFilesByType(Short typeId) {
+      filesTotal += jdbi.onDemand(FilesDao.class).countByTypes(asList(typeId));
```

Comment:
Hier is het gebruik van asList(), zie opm. bij imports

Reply:
7b2bd48